packages/nextra-theme/src/index.tsx

```diff
@@ -30,10 +28,118 @@ import {
 } from '@/components'
 import { DocumentContext, MDXLayoutNav, MDXLayoutOutline, MDXLayoutPagination, NavContext } from '@/layout'
 
+const MDXWrapper: NextraMDXContent = ({ children, toc }) => {
+  const { activePath } = useContext(NavContext)!
+  const { frontMatter, timestamp, readingTime } = useContext(DocumentContext)!
+  const { locale } = useRouter()
+  const lastUpdated = timestamp ? new Date(timestamp) : null
+
+  return (
+    <>
+      <div sx={{ pt: [null, null, null, Spacing['24px']] }}>
+        <div sx={{ display: [null, null, null, 'none'], mb: Spacing['32px'] }}>
+          <MDXLayoutNav mobile />
+        </div>
+
+        <article className="graph-docs-content" sx={mdxStyles}>
```

Comment:
`mdxStyles` is defined below, should it be moved above this component?

Reply:
fixed here https://github.com/graphprotocol/docs/pull/733